app/javascript/src/lib/OWLanguageLinter.js

```diff
@@ -622,3 +623,17 @@ function findUndefinedSubroutines(content) {
     }
   }
 }
+
+function findTripleEquals(content) {
+  const stringRanges = findRangesOfStrings(content);
+
+  for (const match of matchAllOutsideRanges(stringRanges, content, /={3}/g)) {
+    const from = match.index
+    diagnostics.push({
+      from,
+      to: from + match[0].length,
+      severity: "warning",
```

Comment:
It should be an error rather than a warning since it results in an error in-game

Reply:
```suggestion
      severity: "error",
```